.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+This code review checklist is intended to serve as a starting point for the author and reviewer, although it may not be appropriate for all types of changes (e.g. fixing a spelling typo in documentation).  For more in-depth discussion of how we think about code review, please see [Code Review Guidelines](../blob/main/CODE_REVIEW_GUIDELINES.md).
+
+# Author
+<!-- NOTE: Do not modify these when initially opening the pull request.  This is a checklist template that you tick off AFTER the pull request is created. -->
+- [ ] Self-review: Did you review your own code in GitHub's web interface? Code often looks different when reviewing the diff in a browser, making it easier to spot potential bugs.
+- [ ] Automated tests: Did you add appropriate automated tests for any code changes?
+- [ ] Code coverage: Did you check the code coverage report for the automated tests?  While we are not looking for perfect coverage, the tool can point out potential cases that have been missed.
+- [ ] Documentation: Did you update Docs as appropiate? (E.g [README.md](../blob/main/README.md), [LICENSE.md](../blob/main/LICENSE.md), etc.)
```

Comment:
One pain point with GitHub templates—the hyperlinks here will point to the main branch and not the branch under review.  I haven't ever found a variable that allows the PR to refer to the current branch, but if such a variable existed that would be what we really want.

I mention this, because reviewers might click the link and see the version on `main` but not realize that's what they're viewing.

What kinds of ways could this be made less likely to trip people up?

Reply:
Yes, probably license changes won't be frequent

> I mention this, because reviewers might click the link and see the version on main but not realize that's what they're viewing
Yes, this is real. I don't know how to avoid this.